packages/nc-gui/components/cell/Email.vue

```diff
@@ -70,8 +73,10 @@ watch(
     v-if="editEnabled"
     :ref="focus"
     v-model="vModel"
-    class="w-full outline-none text-sm px-1 py-2"
     :placeholder="isEditColumn ? '(Optional)' : ''"
+    class="w-full focus:!outline-none focus:ring-0 focus:border-none !border-none !outline-none text-sm px-2"
```

Comment:
By adding `outline-none`, it should be enough for covering focus case as well. Hence, it seems `focus:!outline-none` is not necessary. When I inspect the input element, I didn't see the case that you need to use `!` to force override.

Reply:
updated